pkg/management/component.go

```diff
@@ -1178,6 +1178,7 @@ func (c *component) GetRealmCustomConfiguration(ctx context.Context, realmName s
 				APISelfAccountDeletionEnabled:       &falseBool,
 				ShowAuthenticatorsTab:               &falseBool,
 				ShowPasswordTab:                     &falseBool,
+				ShowProfileTab:                      &falseBool,
 				ShowMailEditing:                     &falseBool,
```

Comment:
ShowMailEditing should not exist anymore

Reply:
Removed